src/csrc/xsmm_functors.h

```diff
@@ -1286,6 +1286,48 @@ class AddBiasTPP {
   BinaryTPP kernel;
 };
 
+
+template <typename T>
+class AddBias2TPP {
```

Comment:
Should we rename it to RowBCastAddTpp? Also add ColBCastAddTpp and ScalarBCastAddTpp and make them generic with  3 template parameters <Tin0, Tin1, Tout>?

Reply:
removed AddBias2TPP from the latest commit.